project-base/config/packages/shopsys_frontend_api.yml

```diff
@@ -3,8 +3,6 @@ overblog_graphql:
         schema:
             query: Query
             mutation: Mutation
-            resolver_maps:
-                - Shopsys\FrontendApiBundle\Model\Resolver\Products\ProductResolverMap
```

Comment:
I add a point here that the documentation needs to be updated

Reply:
Nice catch! Thanks. fixed in 2ac2d98a9